### PR TITLE
Feature/nightmode

### PR DESCRIPTION
**Pull request by DeepGreen1**

### CODE REVIEW

**Review by pmcintyr — APPROVED** [review]
Good work on this, it seems like you got the work done! A few imports can be removed but the rest is great 🎆🎆🎆

File: app/src/main/java/com/github/brugapp/brug/view_model/SignInViewModel.kt [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+import android.app.Activity
+import android.content.Context
 import android.content.Intent
+import androidx.appcompat.app.AppCompatDelegate
 import androidx.lifecycle.ViewModel
 import com.github.brugapp.brug.data.BrugDataCache
 import com.github.brugapp.brug.data.UserRepository
 import com.github.brugapp.brug.di.sign_in.*
 import com.github.brugapp.brug.di.sign_in.brug_account.BrugSignInAccount
+import com.github.brugapp.brug.ui.SignInActivity
```
[the remark this answers is not in the document]
Comment (DeepGreen1) [reply app/src/main/java/com/github/brugapp/brug/view_model/SignInViewModel.kt:12]: I'll try to add some tests so I'll do it, thanks for spotting it!

File: app/src/main/java/com/github/brugapp/brug/ui/SignInActivity.kt [hunk 2]
```diff
@@ -48,6 +48,8 @@ class SignInActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_sign_in)
 
+        viewModel.checkNightMode(this)
```
[the remark this answers is not in the document]
Comment (DeepGreen1) [reply app/src/main/java/com/github/brugapp/brug/ui/SignInActivity.kt:51]: I'll do the same for this

**Review by omarelmalki — COMMENTED** [review]
.

**Review by omarelmalki — APPROVED** [review]
Great job on the night mode feature !

**Pull request merged**